lib/fabrik/DirectorInstance.js

```diff
@@ -50,37 +67,62 @@ class DirectorInstance extends BaseInstance {
         this.networkSegmentIndex = networkSegmentIndex;
       })
       .tap(() => {
+        //TODO: This still has coupling with space guid as it calls ensureTenantId. 
```

Comment:
Please remove this comment. It is obsolete now.

Reply:
sure